pyproject.toml

```diff
@@ -1,11 +1,11 @@
 [tool.poetry]
-name = "software-gardening-almanac"
-version = "0.0.1"
+name = "almanack"
+version = "0.0.0"
```

Comment:
why is this bump down?

Reply:
Thanks for the double check here, the earlier version entry was I believe prior to dynamic versioning capabilities and this is a correction towards that effect (where `0.0.0` is the preference via `"Poetry still requires the tool.poetry.version field to be present in pyproject.toml, but you are encouraged to use version = "0.0.0" as a standard placeholder."` from [here](https://github.com/mtkennerly/poetry-dynamic-versioning)).